packages/amplify-migration-tests/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "amplify-migration-tests",
+  "version": "2.13.3",
+  "description": "",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/aws-amplify/amplify-cli.git",
+    "directory": "packages/amplify-migration-tests"
+  },
+  "author": "Amazon Web Services",
+  "license": "Apache-2.0",
+  "main": "index.js",
+  "keywords": [
+    "graphql",
+    "appsync",
+    "aws"
+  ],
+  "private": true,
+  "scripts": {
+    "migration": "npm run setup-profile latest && jest",
+    "e2e": "npm run setup-profile && jest",
```

Comment:
why do we need this? 
"npm run setup-profile" is the same as "npm run setup-profile latest" because the default version is "latest", 
so "e2e" is the same as "migration".

Reply:
removed e2e script